engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -219,6 +219,35 @@ paths:
       security:
         - google_oauth:
             - openid
+ '/workflows/${version}/{id}/addLabels':
```

Comment:
I'm thinking this endpoint should just be called `labels`, with the verb implied by the HTTP POST.